docs/typedef/modules/controllers.js

```diff
@@ -0,0 +1,18 @@
+/*
+* Copyright © 2018 Lisk Foundation
+*
+* See the LICENSE file at the top-level directory of this distribution
+* for licensing information.
+*
+* Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+* no part of this software, including this file, may be copied, modified,
+* propagated, or distributed except according to the terms contained in the
+* LICENSE file.
+*
+* Removal or modification of this copyright notice is prohibited.
+*/
+/**
+ * Functional Namespace `controllers`
+ *
+ * @namespace controllers
```

Comment:
Would it actually make more sense to locate this definition in an index file in `api/controllers`? And/or to call it `api/controllers` rather than just `controllers`?

Reply:
I'm fine with api/controllers.

If we change the location of the file to the files it refers to, would it apply also to the files in typedef/modules?